types/jest-global.d.ts

```diff
@@ -1,7 +1,5 @@
 /// <reference types="./standalone.js"/>
 
-declare const expect: ExpectWebdriverIO.Expect
```

Comment:
What is the purpose of this removal? Seems unrelated to the changeset.

Reply:
<img width="522" alt="image" src="https://github.com/webdriverio/expect-webdriverio/assets/19924396/4b9f68ab-2aab-434e-9081-d12683653e15">
this conflict again @types/jest/index.d.ts

Reply:
the ExpectWebdriverIO.expect has been updated in expect-webdriverio.d.ts file